shared/common-adapters/text.native.js

```diff
@@ -90,6 +106,9 @@ class Text extends Component<Props> {
       // make a dummy one so that it shows the selection (on iOS).
       (this.props.selectable ? () => {} : undefined)
 
+    const onLongPress =
+      this.props.onLongPress || (this.props.onLongPressURL ? this._urlChooseOption : undefined)
```

Comment:
This is new, it lets this text component be shared between desktop/mobile better